api/Cargo.toml

```diff
@@ -21,9 +24,11 @@ lazy_static = { version = "1.4.0", optional = true }
 mio = { version = "0.8", features = ["os-poll", "os-ext"], optional = true }
 serde = { version = "1.0.110", features = ["rc", "serde_derive"] }
 url = { version = "2.1.1", optional = true }
+tokio = { version = "1.24", features = ["full"] }
```

Comment:
It seems the using of `full` feature will import a large number of dependencies, can we only use the necessary feature?